tests/conftest.py

```diff
@@ -69,6 +69,16 @@ def inner(tz='CET'):
     return inner
 
 
+@pytest.fixture
+def blank_todo(default_database):
```

Comment:
I think it's more generally usable if this fixture returns a function that creates todos (eventually with a parameter for specifying list)